projects.json

```diff
@@ -2572,6 +2572,44 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/louisdh/cub.git",
+    "path": "cub",
```

Comment:
We like to keep projects sorted by the "path" field. Would you mind moving the whole cub project blob so that it's after the CoreStore project?